### PR TITLE
NCC Extravagant Replication

### PR DESCRIPTION
**Pull request by TheBlackMarvel**

### CODE REVIEW

**Comment by TheBlackMarvel** [comment]
Hmm, think I've screwed up here. All I wanted to do was add the card...

**Comment by Hanmac** [comment]
try to `git rebase origin/master -i`

File: forge-gui/res/cardsfolder/upcoming/extravagant_replication.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Extravagant Replication
+ManaCost:4 U U
+Types:Enchantment
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigClone | TriggerDescription$ At the beginning of your upkeep, create a token that's a copy of another target nonland permanent you control.
+SVar:TrigClone:DB$ CopyPermanent | ValidTgts$ Permanent.nonLand+YouCtrl | TgtPrompt$ Select target non-land permanent you control. | NumCopies$ 1
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/extravagant_replication.txt:5]: needs `+Other`

**Comment by TheBlackMarvel** [comment]
> try to `git rebase origin/master -i`

Alas, this is definitely beyond my abilities. Tried looking it up and all it tells me is I'm _definitely_ not using Github the way it's supposed to be used.

**Review by Hanmac — APPROVED** [review]
👍

**Pull request merged**